<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="admin" failures="2" tests="25" name="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" time="413.167" errors="2" timestamp="2019-09-26T11:38:09 IST" skipped="14">
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyAddCancelButton" time="25.368"/>
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyAddRecordwithoutFunctionality" time="32.438"/>
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyAddRecordWithoutHotLine" time="37.739"/>
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyAddRecordWithoutStartDateTime" time="33.413"/>
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyAddRecordWithoutEndDateTime" time="34.695"/>
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyAddRecordWithoutLanguage" time="32.105"/>
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyAddRecordWithoutWaveFile" time="35.233"/>
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="AddRevertRecord" time="64.129">
    <error message="no such element: Unable to locate element: {&quot;method&quot;:&quot;css selector&quot;,&quot;selector&quot;:&quot;.toast-message&quot;}
  (Session info: chrome=77.0.3865.90)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;ADMIN&#039;, ip: &#039;192.168.1.15&#039;, os.name: &#039;Windows 8.1&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.3&#039;, java.version: &#039;9.0.4&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 77.0.3865.90, chrome: {chromedriverVersion: 76.0.3809.68 (420c9498db8ce..., userDataDir: C:\Users\ADMINP~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:57029}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: b0b8b8bac0e0f4d726d78ff1b838adb5
*** Element info: {Using=css selector, value=.toast-message}" type="org.openqa.selenium.NoSuchElementException">
      <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":".toast-message"}
  (Session info: chrome=77.0.3865.90)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'ADMIN', ip: '192.168.1.15', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '9.0.4'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 77.0.3865.90, chrome: {chromedriverVersion: 76.0.3809.68 (420c9498db8ce..., userDataDir: C:\Users\ADMINP~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:57029}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: b0b8b8bac0e0f4d726d78ff1b838adb5
*** Element info: {Using=css selector, value=.toast-message}
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.base/java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByCssSelector(RemoteWebDriver.java:420)
at org.openqa.selenium.By$ByCssSelector.findElement(By.java:431)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy12.isDisplayed(Unknown Source)
at com.tetherfi.pages.IntroMessageAnnouncementPage.getSuccessMessage(IntroMessageAnnouncementPage.java:956)
at com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest.AddRevertRecord(IntroMessageAnnouncementCreateTest.java:164)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- AddRevertRecord -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyRevertForAddNewRecord" time="45.618">
    <error message="Expected condition failed: waiting for element to be clickable: [[[[[[[[ChromeDriver: chrome on XP (e23108aa495678268d45209344903511)] -&gt; id: grid]] -&gt; tag name: tr]] -&gt; tag name: td]] -&gt; id: isEnabled] (tried for 20 second(s) with 500 milliseconds interval)
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;ADMIN&#039;, ip: &#039;192.168.1.15&#039;, os.name: &#039;Windows 8.1&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.3&#039;, java.version: &#039;9.0.4&#039;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 77.0.3865.90, chrome: {chromedriverVersion: 76.0.3809.68 (420c9498db8ce..., userDataDir: C:\Users\ADMINP~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:57062}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: e23108aa495678268d45209344903511" type="org.openqa.selenium.TimeoutException">
      <![CDATA[org.openqa.selenium.TimeoutException: Expected condition failed: waiting for element to be clickable: [[[[[[[[ChromeDriver: chrome on XP (e23108aa495678268d45209344903511)] -> id: grid]] -> tag name: tr]] -> tag name: td]] -> id: isEnabled] (tried for 20 second(s) with 500 milliseconds interval)
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'ADMIN', ip: '192.168.1.15', os.name: 'Windows 8.1', os.arch: 'amd64', os.version: '6.3', java.version: '9.0.4'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 77.0.3865.90, chrome: {chromedriverVersion: 76.0.3809.68 (420c9498db8ce..., userDataDir: C:\Users\ADMINP~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:57062}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: XP, platformName: XP, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify}
Session ID: e23108aa495678268d45209344903511
at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:95)
at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:272)
at com.tetherfi.pages.BasePage.waitUntilWebElementIsClickable(BasePage.java:252)
at com.tetherfi.pages.BasePage.selectWebElement(BasePage.java:192)
at com.tetherfi.pages.IntroMessageAnnouncementPage.selectRecord(IntroMessageAnnouncementPage.java:1472)
at com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest.VerifyRevertForAddNewRecord(IntroMessageAnnouncementCreateTest.java:171)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- VerifyRevertForAddNewRecord -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyApprovedDataSectionWithoutApproval" time="20.307">
    <skipped/>
  </testcase> <!-- VerifyApprovedDataSectionWithoutApproval -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyAuditTrailReportForRevert" time="19.650">
    <skipped/>
  </testcase> <!-- VerifyAuditTrailReportForRevert -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="AddRejectRecord" time="16.537">
    <failure message="expected [Record created successfully] but found [Duplicate Record]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Record created successfully] but found [Duplicate Record]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest.AddRejectRecord(IntroMessageAnnouncementCreateTest.java:206)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- AddRejectRecord -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifySendForApprovalForAddNewRecord" time="0.000">
    <skipped/>
  </testcase> <!-- VerifySendForApprovalForAddNewRecord -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="RejectforAddNewIntroMessageAnnouncementRecord" time="0.000">
    <skipped/>
  </testcase> <!-- RejectforAddNewIntroMessageAnnouncementRecord -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyAuditTrailReportForReject" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyAuditTrailReportForReject -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyRecordAfterRejection" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyRecordAfterRejection -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="AddNewIntroMessageAnnouncementRecord" time="15.934">
    <failure message="expected [Record created successfully] but found [Duplicate Record]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Record created successfully] but found [Duplicate Record]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest.AddNewIntroMessageAnnouncementRecord(IntroMessageAnnouncementCreateTest.java:256)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.base/java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- AddNewIntroMessageAnnouncementRecord -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyAuditTrailReportForCreate" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyAuditTrailReportForCreate -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyAuditTrailDataForAddNewIntroMessageAnnouncementRecord" time="0.001">
    <skipped/>
  </testcase> <!-- VerifyAuditTrailDataForAddNewIntroMessageAnnouncementRecord -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifySendForApprovalForAddNewIntroMessageAnnouncementRecord" time="0.000">
    <skipped/>
  </testcase> <!-- VerifySendForApprovalForAddNewIntroMessageAnnouncementRecord -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyAuditTrailReportForSendForApproval" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyAuditTrailReportForSendForApproval -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="ApproveforAddNewIntroMessageAnnouncementRecord" time="0.000">
    <skipped/>
  </testcase> <!-- ApproveforAddNewIntroMessageAnnouncementRecord -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyAuditTrailReportForApprove" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyAuditTrailReportForApprove -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="VerifyRecordApprovedDataSectionafterApproval" time="0.000">
    <skipped/>
  </testcase> <!-- VerifyRecordApprovedDataSectionafterApproval -->
  <testcase classname="com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest" name="AddDuplicateRecord" time="0.000">
    <skipped/>
  </testcase> <!-- AddDuplicateRecord -->
</testsuite> <!-- com.tetherfi.test.ivr.IntroMessageAnnouncementCreateTest -->
